src/universal/modules/meeting/components/MeetingUpdatesPrompt/MeetingUpdatesPrompt.js

```diff
@@ -1,36 +1,54 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import MeetingPrompt from 'universal/modules/meeting/components/MeetingPrompt/MeetingPrompt';
 import {createFragmentContainer} from 'react-relay';
+import HelpTextForTeam from 'universal/modules/meeting/components/HelpTextForTeam';
+import HelpTextMyRound from 'universal/modules/meeting/components/HelpTextMyRound';
+import MeetingPrompt from 'universal/modules/meeting/components/MeetingPrompt/MeetingPrompt';
 
 const MeetingUpdatesPrompt = (props) => {
   const {
+    agendaInputRef,
     localPhaseItem,
-    team: {teamMembers}
+    team
   } = props;
+  const {meetingNumber, teamMembers} = team;
   const currentTeamMember = teamMembers[localPhaseItem - 1];
-  const heading = <span>{currentTeamMember.preferredName}, <i>what’s changed since our last meeting</i>?</span>;
+  const {isSelf: isMyMeetingSection} = currentTeamMember;
+  const isFirstMeeting = meetingNumber === 1;
```

Comment:
@mattkrick we discussed changing the logic from “assumed first meeting” to whether or not the user has projects on the board…I’ve had some trial and error trying to update the GraphQL fragments for Relay…looking at `MeetingContainer` and `MeetingUpdates` for the viewer, projects. Mind making a commit here? Is it expensive to add viewer/projects for the update, too?

Reply:
@mattkrick I would like to know about projects instead of meeting number in the updates prompt. I was a bit lost in trial-and-error trying to update the query.